src/util/settings/connection_client.cc

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2017 CodiLime
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include <QSettings>
+
+#include "util/settings/connection_client.h"
+
+namespace veles {
+namespace util {
+namespace settings {
+namespace connection {
+
+bool default_run_server = true;
+QString localhost("127.0.0.1");
+QString default_database("veles.vdb");
+int default_server_port = 3135;
+bool default_shut_down_server = true;
+QString settings_true("yes, please");
```

Comment:
these values "yes, pelase" and "no, thanks" looks like something that we want to display in UI, but not save  into settings